### ISSUE TITLE
[ENH] A way to import and export the settings

### ISSUE THREAD

**Issue by aragaer** [org-roam/org-roam-ui#223]
**Is your feature request related to a problem? Please describe.**
I have a desktop and a laptop and I wanted to use ORUI on both. I spent some time setting up the ORUI on the desktop and then I found that I don't remember what exactly I tweaked to get the behavior I got which I would like to get working on the laptop.

**Describe the solution you'd like**
Just having a textarea with the current settings (as a JSON?) so that I could copy that and then paste in a different browser would be enough.

**Comment by ThomasFKJorna** [comment]
Yeah this really is a long time coming, I will try to add this soon!

**Comment by chris-aeviator** [comment]
Isn't https://github.com/org-roam/org-roam-ui/issues/231 more feasible  in that regard? Saving the json within the modules folder (sent over using websocket) should work just fine?
